### PR TITLE
69 wrong last move

### PR DESCRIPTION
**Pull request by ghUserrrr**
now the cheater sees the "end the game" button
both players can end the game
cheater's screen: 

Image

Image

### CODE REVIEW

File: pages/games/[gameType].tsx [hunk 1]
```diff
@@ -316,7 +318,14 @@ const Game: NextPage<IGamePageProps> = ({ gameType, version }) => {
       message: { gameId, disputeRunner: account.address! }
     })
 
-    const finishedGameResult = await disputeMove(await getArbiter(), gameState.lastOpponentMove);
+    let lastMove = gameState.lastOpponentMove;
```
Comment (utgarda) [inline pages/games/[gameType].tsx:321]: Please introduce a name for this check and that helps you eliminate reassigning `lastMove`:
```typescript
const aReadableNameForThisCheckResult = lastNonce > lastOpponentNonce && gameState.lastMove !== null
const lastMove = aReadableNameForThisCheckResult ? gameState.lastOpponentMove : lastMove;
```

**Review by utgarda — CHANGES_REQUESTED** [review]
Looks good. Please take care of message wordings.

File: pages/games/[gameType].tsx (hunk 1 above)
Comment (ghUserrrr) [reply pages/games/[gameType].tsx:321]: запишу 
opponentsOrOwnMoveToSend

Comment (ghUserrrr) [reply pages/games/[gameType].tsx]: done

**Review by poludnev — APPROVED** [review]

**Review by utgarda — APPROVED** [review]

**Pull request merged**